iocore/net/SSLUtils.cc

```diff
@@ -2034,7 +2034,12 @@ SSLConnect(SSL *ssl)
 
       Debug("ssl.origin_session_cache", "origin session cache lookup key = %s", lookup_key.c_str());
 
-      sess = origin_sess_cache->get_session(lookup_key);
+      TLSSessionResumptionSupport *srs = TLSSessionResumptionSupport::getInstance(ssl);
```

Comment:
Adding these lines here is actually a bit of sad thing, because `SSLConnect` was originally independent from NetVC. You could use it for any SSL connections. `SSLReadBuffer` too, before supporting early data. I think that's the reason those functions are in SSLUtils.

Now those two functions depend on NetVC, and there is no longer a reason to have those in SSLUtils as non-member functions. Although I don't think we should do this on this PR, we should probably make the two functions (and SSLWriteBuffer) member functions of SSLNetVC and move them to P_SSLNetVConnection.h. Then we don't need to call `getInstance` to call `getOriginSession`.

Reply:
I agree, was trying to stay away from this, but looks like we'll have to do things like this for now.